lib/relay.js

```diff
@@ -169,7 +169,7 @@ function responseHandler(filterRules, config, io) {
 
     filters({ url, method, body, headers }, (error, result) => {
       if (error) {
-        const reason = 'Response does not match any accept rule, blocking websocket request';
+        const reason = 'Request does not match any accept rule, blocking websocket request';
```

Comment:
this was not a response and confused me 😕

Reply:
Please undo this change, this is indeed a response. The naming is a bit convoluted because the relay logic applies both to the server and the client. A response in this case is the answer being relayed back to the entity that ‘asked the question’, and will be dropped if it isn’t filtered.